CHANGELOG.md

```diff
@@ -0,0 +1,170 @@
+# Changelog
+All notable changes to the SpatialOS Game Development Kit for Unreal will be documented in this file.
+
+The format of this Changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - 2019-xx-xx
+
+### Breaking Changes:
+- The GDK now uses SpatialOS 13.6.2. You **must** change the version specified in your projects `spatialos.json` file.
```

Comment:
> You **must** change

Isn't that version number unused?

Reply:
Good spot, I've removed this call to action.